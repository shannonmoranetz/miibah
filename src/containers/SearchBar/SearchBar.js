import React, { Component } from 'react';

class SearchBar extends Component {
  render() {
    return (
      <div className="SearchBar">
        <form className="search-form">
          <input className="search-input" placeholder="search for an amiibo..."/>
          <input className="submit-form" type="submit"/>
        </form>
      </div>
    );
  }
}

export default SearchBar