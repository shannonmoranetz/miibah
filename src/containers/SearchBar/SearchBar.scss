.SearchBar {
  display: flex;
  justify-content: center;
}

textarea:focus, input:focus{
  outline: none;
}

.search-form {
  display: flex;
  margin: 0px;
  padding: 10px;
}

.search-input {
  font-size: 16px;
  width: 300px;
  font-family: $body-font;
  padding: 8px;
  border: 3px solid $darkblue;
  border-radius: 15px;
}

.submit-form {
  padding: 10px;
  border: 1px solid $deepblue;
  font-size: 16px;
  margin-left: 30px;
  width: 20%;
  font-family: $heading-font;
  border: 2px solid white;
  background-color: $salmon;
  color: white;
  border-radius: 15px;
  cursor: pointer;
}

.submit-button {
  font-family: $heading-font;
  margin: 4px 12px;
  text-align: center;
  padding: 7px;
  font-size: 18px;
  color: $deepblue;
}

.submit-button:hover {
  color: tomato;
}