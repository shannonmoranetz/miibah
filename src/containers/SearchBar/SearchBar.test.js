import React from 'react';
import SearchBar from './SearchBar';
import { shallow } from 'enzyme';

describe('SearchBar', () => {
  let wrapper; 
  beforeEach(() => {
    wrapper = shallow(<SearchBar />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});