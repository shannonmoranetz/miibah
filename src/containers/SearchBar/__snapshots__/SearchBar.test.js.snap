// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should match the snapshot 1`] = `
<div
  className="SearchBar"
>
  <form
    className="search-form"
  >
    <input
      className="search-input"
      placeholder="search for an amiibo..."
    />
    <input
      className="submit-form"
      type="submit"
    />
  </form>
</div>
`;
